APIBacklog/documentation/APIBacklog.md

```diff
@@ -3,51 +3,61 @@
+| Device Location | DTAG | 2022/01/11 | N/A | DTAG, Orange, Telefonica, Vodafone | Approved (2022/04/17) | [DeviceLocation](https://github.com/camaraproject/DeviceLocation) | location-verification | [Link](https://github.com/camaraproject/DeviceLocation/blob/main/documentation) | v0.2.0-wip | [YAML](https://github.com/camaraproject/DeviceLocation/blob/main/code/API_definitions/location-verification.yaml) | N/A | N/A | N/A|
+| |  |  |  |  |   | |  |  | v0.1.0 | [YAML](https://github.com/camaraproject/DeviceLocation/blob/release-v0.1.0/code/API_definitions/location.yaml) | N/A | N/A | N/A|%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%<!---Device Status-->%%%%%%%%%%%%%%%%%%%%%
+| Device Status | DTAG | 2022/01/11 | N/A | DTAG, Orange, Telefonica, Vodafone | Approved (2022/04/17) |  [DeviceStatus](https://github.com/camaraproject/DeviceStatus) | device-status | [Link](https://github.com/camaraproject/DeviceStatus/tree/main/documentation) | v0.5.0-wip | [YAML](https://github.com/camaraproject/DeviceStatus/blob/main/code/API_definitions/device-status.yaml) |  N/A |  N/A |  N/A|
+| |  |  |  |  |  |   |  |  | v0.4.1 |  N/A | N/A | N/A | N/A|%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%<!---Home Devices Quality on Demand --->
+| Home Devices Quality on Demand | Telefonica | 2022/09/14 | [Link](https://github.com/camaraproject/WorkingGroups/blob/main/APIBacklog/documentation/SupportingDocuments/API%20proposals/APIproposal_HomeDevicesQoD_Telefonica.md) | DTAG, KDDI, Telefonica |  Approved (2022/10/13) | [HomeDevicesQoD](https://github.com/camaraproject/HomeDevicesQoD) | home_devices_qod | [Link](https://github.com/camaraproject/HomeDevicesQoD/blob/main/documentation/API_documentation) | v0.2.0 | [YAML](https://github.com/camaraproject/HomeDevicesQoD/blob/main/code/API_definitions/home_devices_qod.yaml) |  N/A |  N/A |  N/A|
+| | |  | | | | | | | v0.1.0 | [YAML](https://github.com/camaraproject/HomeDevicesQoD/blob/release-0.1.0/code/API_definitions/home_devices_qod.yaml) |  N/A |  N/A |  N/A|<!---SIMSwap-->%%%%%%%%%%%%%%%%%%%%%
+| SIM Swap | DTAG | 2022/09/30 | [Link](https://github.com/camaraproject/WorkingGroups/blob/main/APIBacklog/documentation/SupportingDocuments/API%20proposals/APIproposal_Sim%20Swap_DeustcheTelekom.md) | Airtel, DTAG, GSMA-IDG, KDDI, Teelfonica | Approved (2022/10/13) | [SIMSwap](https://github.com/camaraproject/SimSwap) | sim_swap | [Link](https://github.com/camaraproject/SimSwap/tree/main/documentation) | v0.4.1 | [YAML](https://github.com/camaraproject/SimSwap/blob/main/code/API_definitions/sim_swap.yaml) | N/A | N/A | N/A|N/A|%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%<!---Number Verification-->%%%%%%%%%%%%%%%%%%%%%
```

Comment:
There is no released or tagged version ... the link into main points in the meantime to 0.5.0-wip